### PR TITLE
Cleanup ObjectCheck

### PR DESCRIPTION
**Pull request by dmitripivkine**
Update ObjectCheck.cpp and ObjecrCheck.hpp according Coding Standards:
- use correct variable types
- fix code formatting
- replace extensions to ext

### CODE REVIEW

**Comment by dmitripivkine** [comment]
fixes https://github.com/eclipse-openj9/openj9/issues/20468

File: runtime/gc_base/ObjectCheck.cpp [hunk 1]
```diff
@@ -50,7 +50,7 @@ isValidClass(J9JavaVM *javaVM, J9Class *ptr, UDATA flags) {
 	}
 
 	/* check alignment */
-	if (0 != ((UDATA)ptr & (sizeof(UDATA) - 1))) {
+	if (0 != ((uintptr_t)ptr & (sizeof(uintptr_t) - 1))) {
```
Comment (amicic) [inline runtime/gc_base/ObjectCheck.cpp:53]: while we can keep this change minimalistic (formatting only), we should probably use a proper const/variable rather then sizeof(uintptr_t)

we objectAlignmentInBytes available at a few places 

although I'm a bit confused - ptr is pointer to RAM class, should we not have a strong check for 256 byte (for what we have J9_REQUIRED_CLASS_ALIGNMENT)!?

Comment (dmitripivkine) [reply runtime/gc_base/ObjectCheck.cpp:53]: Yes, it caught my eye too, we need to check correctness as well is this code in use or obsolete. I added this to my todo list. I think it is better to keep this PR formatting only.
Regarding `sizeof(uintptr_t)` I thought about this too. Do we have better declaration?

**Comment by amicic** [comment]
jenkins compile win,aix jdk21

**Review by amicic — APPROVED** [review]

**Pull request merged**